stylesheets/_modules.scss

```diff
@@ -156,6 +161,7 @@
   flex-direction: row;
   align-items: flex-start;
   border: 1px solid $color-black-015;
+  flex-grow: 1;
```

Comment:
Just me preference but I like to group CSS properties together. So flex-grow could be after align-items in this case.

Reply:
:+1: